client/packages/system/src/Item/context/useUnitVariant.ts

```diff
@@ -98,14 +86,14 @@ export const useUnitVariant = (
   variantsControl?: VariantControl;
   unitVariantsExist: boolean;
 } => {
-  const [item, userSelectedVariantId, setUserSelectedVariant] = useUnitStore(
-    state => [
-      state.items[itemId],
-      state.userSelectedVariants[itemId],
-      state.setUserSelectedVariant,
-    ],
-    isEqual
+  const authContext = useAuthContext();
+  const [userSelectedVariants, setUserSelectedVariant] = useLocalStorage(
+    `/user/${authContext.user?.id ?? '?'}/store/${
```

Comment:
In the the admin user id is an empty string. Can't remember why and haven't investigated...

Reply:
Just update line 142 in `useLogin` to 
```typescript
        id: userDetails?.userId ?? '',
```